plugins/BEdita/Core/tests/TestCase/Model/Table/AuthProvidersTableTest.php

```diff
@@ -88,15 +89,17 @@ public function validationProvider()
     {
         return [
             'valid' => [
-                true,
+                [],
                 [
                     'name' => 'some_unique_value',
                     'url' => 'https://example.com/oauth2',
                     'params' => [],
                 ],
             ],
             'notUnique' => [
-                false,
+                [
+                    'name.unique',
```

Comment:
Now we are using `$authProvider = $this->AuthProviders->newEntity([]);` n this test case (Cake 4 compatibility) validation output differs

```suggestion
                    'name._required'
                    'name.unique'
```